spec/rubocop/cop/style/symbol_proc_spec.rb

```diff
@@ -293,6 +293,15 @@
         end
       RUBY
 
+      expect_no_offenses(<<~RUBY)
+        something
+
+        something do |e|
+          # comment
+          e.upcase
+        end
+      RUBY
```

Comment:
Looks good! Can you also add a test for `numblock` somewhere below L350 in this spec file?